plugins/usbdmx/AsyncPluginImpl.cpp

```diff
@@ -175,6 +183,14 @@ bool AsyncPluginImpl::NewWidget(JaRuleWidget *widget) {
                                 new JaRuleDevice(m_plugin, widget, str.str()));
 }
 
+bool AsyncPluginImpl::NewWidget(NodleU1 *widget) {
+  return StartAndRegisterDevice(
+      widget,
+      new NodleU1Device(m_plugin, widget, "DMXControl Projects e.V. Nodle U1",
```

Comment:
The serial number is already shown in the "Description" field like (IIRC) the Enttec plugin does. Is that not okay? I think the description is visible everywhere you see the device name in the UI...